main.css

```diff
@@ -25,6 +25,13 @@ table, th , td{
 th, td{
     text-align: left;
 }
+
 .BoardBlock{
     height: 80px;
+}
+
+.blockDimension{
```

Comment:
Is there a reason for camelCase? we should stay consistent cause the other ones are Pascal

Reply:
no reason, it's okay to change it back